meta-iot2000-example/recipes-example/iot2000setup/files/expandfs.sh

```diff
@@ -1,31 +1,29 @@
 #!/bin/sh
 
-set -eu
+THIS_SCRIPT="$(basename $0)"
 
-PART_NUMBER=3
+log () {
+	echo "${THIS_SCRIPT}: $1" > /dev/kmsg
```

Comment:
Below, you use `logger`. Why not also here or just one of both?

Reply:
The step to be done is logged in the boot log and the detailed output of the programs is logged with logger. I thought this is a good idea to separate them for cleaner kernel logs.

Reply:
Hm, seems inconsistent to me to have to look in 2 places, particularly as this script is intended to run once. I'd prefer to have this in one central place.

Reply:
okay